src/wallet/components/WalletModal.tsx

```diff
@@ -0,0 +1,267 @@
+import { useConnect, useConnectors } from 'wagmi';
+import { defaultAvatarSVG } from '../../internal/svg/defaultAvatarSVG';
+import {
+  background,
+  border,
+  cn,
+  color,
+  line,
+  pressable,
+  text,
+} from '../../styles/theme';
+import { useOnchainKit } from '../../useOnchainKit';
+
+const BaseIcon = () => (
```

Comment:
can we move these SVGs to their own files

Reply:
Yep! Sorry, I was planning to. This PR wasn't really ready for review, just ready for component QA!